pkgs/by-name/fi/field-monitor/package.nix

```diff
@@ -0,0 +1,102 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  cargo,
+  meson,
+  ninja,
+  rustPlatform,
+  rustc,
+  pkg-config,
+  glib,
+  gsettings-desktop-schemas,
+  gtk4,
+  libadwaita,
+  libvirt,
+  gst_all_1,
+  desktop-file-utils,
+  appstream,
+  appstream-glib,
+  wrapGAppsHook4,
+  xdg-desktop-portal,
+  blueprint-compiler,
+  libxml2,
+  spice-protocol,
+  spice-gtk,
+  vte-gtk4,
+  gtk-vnc,
+  usbredir,
+  libepoxy,
+  libGL,
+  openssl,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "field-monitor";
+  version = "49.0";
+
+  strictDeps = true;
+
+  src = fetchFromGitHub {
+    owner = "theCapypara";
+    repo = "field-monitor";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-hKxYwYcUClBfHhi4sfNCZBVADXPRKQxHUHZS6l7BV+A=";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoVendor {
+    inherit (finalAttrs) src;
+    hash = "sha256-sd+yQLEqMnNNEYofPCd8wXJeNh0TYBS/E9FzrzfGEsk=";
+  };
+
+  mesonBuildType = "release";
+
+  postInstall = ''
+    wrapProgram $out/bin/de.capypara.FieldMonitor --prefix PATH ':' "$out/libexec"
+  '';
+
+  nativeBuildInputs = [
+    appstream
+    appstream-glib
```

Comment:
Interesting that it works with `strictDeps`, but these and `libxml2` (unless you use `bin/{xmlcatalog,xmllint}` at build time?) are probably used for linking as libraries, in which case you want these for the host (not build) platform, and so in `buildInputs`.